config/locales/en.yml

```diff
@@ -54,16 +54,28 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    admin_hidden_work_notification:
+      subject: "[%{app_name}] Your work has been hidden by the Abuse Team"
+      hello: "Dear %{user},"
+      tos: "Terms of Service"
+      abuse: "contact our Abuse Team"
+      sincerely: "Sincerely,"
+      ao3_abuse: "AO3 Abuse"
+      part2: "We are investigating the matter and will contact you once the investigation has been completed. While the work is under investigation it has been hidden; only you will be able to access it."
+      html:
+        part1: "Your work <i>%{title}</i> has been reported as being in violation of the Archive of Our Own's %{tos}."
+        part3: "If you have any questions, please %{abuse}."
+      text:
+        part1: "Your work %{title} has been reported to the Abuse Team as a violation of the Archive of Our Own's Terms of Service (%{tos_url})"
```

Comment:
There should be a period at the end of the sentence here, after the `)`. Also, in the existing email, the work title is in quotation marks and has the URL after at, like: `Your work "TITLE" (http://ao3.org/works/####)`.

ALSO, the phrasing of these two emails is inconsistent. /o\ I checked with Abuse, and could you change `part1` of the plain text email so the whole thing says: 

`Your work "%{title}" (%{work_url}) has been reported as being in violation of the Archive of Our Own's Terms of Service (%{tos_url}).`

That way, except for the necessary adjustments for links, the two emails actually give the same information with the same phrasing.